xla/stream_executor/gpu/gpu_blas_lt.h

```diff
@@ -182,20 +181,19 @@ struct BlasLt {
         DeviceMemoryBase aux,   // may be null
         DeviceMemoryBase a_scale, DeviceMemoryBase b_scale,
         DeviceMemoryBase c_scale, DeviceMemoryBase d_scale,
-        DeviceMemoryBase d_amax, const MatmulAlgorithm& algorithm,
-        DeviceMemoryBase workspace,
+        DeviceMemoryBase d_amax, DeviceMemoryBase workspace,
         blas::ProfileResult* profile_result = nullptr) const {
       return ExecuteOnStream(
-          stream, algorithm,
+          stream, 
           MemoryArgs{a, b, c, d, bias, aux, a_scale, b_scale, c_scale, d_scale,
                      d_amax, workspace, nullptr},
           profile_result);
     }
 
     // The most general form: to be implemented by derived clases.
     virtual absl::Status ExecuteOnStream(
-        Stream* stream, const MatmulAlgorithm& algorithm,
-        const MemoryArgs& args, blas::ProfileResult* profile_result) const = 0;
+        Stream* stream, const MemoryArgs& args, 
+        blas::ProfileResult* profile_result) const = 0;
```

Comment:
Can you make a shim with the original overload that calls `SetAlgorithm` and then this new function, so that it doesn't break TF? Afterwards we can merge the TF change and then remove the shim.